src/main/java/at/ac/tuwien/kr/alpha/Alpha.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2017-2019, the Alpha Team.
+ * Copyright (c) 2017-2018, the Alpha Team.
```

Comment:
We should keep that at 2019. :-)

Reply:
I went through the code now and tried to fix up all the copyright notices I could find, should be better now